packages/@aws-cdk/aws-ecs/lib/ec2/ec2-service.ts

```diff
@@ -254,7 +279,7 @@ export class Ec2Service extends BaseService implements IEc2Service {
  * Validate combinations of networking arguments
  */
 function validateNoNetworkingProps(props: Ec2ServiceProps) {
-  if (props.vpcSubnets !== undefined || props.securityGroup !== undefined || props.assignPublicIp) {
+  if (props.vpcSubnets !== undefined || props.securityGroups !== undefined || props.assignPublicIp) {
     throw new Error('vpcSubnets, securityGroup and assignPublicIp can only be used in AwsVpc networking mode');
```

Comment:
nit: securityGroups

Reply:
Does changing the error message count as a breaking change? if a user explicitly catches this error and checks for the message.

Reply:
I changed it, but let me know of your experience with the comment above

Reply:
I wouldn't consider an error message as a breaking change. If an error message cn change even slightly I don't think it's reasonable to expect the error message thrown by errors to never change.